nexus/tests/integration_tests/instances.rs

```diff
@@ -4175,7 +4172,10 @@ async fn test_instance_ephemeral_ip_no_default_pool_error(
     };
     let error =
         object_create_error(client, &url, &body, StatusCode::NOT_FOUND).await;
-    assert_eq!(error.message, msg);
+    assert_eq!(
+        error.message,
+        "not found: ip-pool with name \"nonexistent-pool\""
+    );
```

Comment:
This used to complain about the default pool despite another pool being specified because the SNAT IP came from the default pool, and that failed before any ephemeral IP stuff came up.